applications/nublado/templates/controller-ingress-files.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: "nublado-controller-files"
+  labels:
+    {{- include "nublado.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  scopes:
+    all:
+      - "exec:notebook"
+  delegate:
+    notebook: {}
```

Comment:
This is much too much power for the fileserver. You only need an internal token with no scopes.

Reply:
...don't I need a regular user token with scope "exec:notebook" ?  Just leave off the delegate: stanza, right?

Reply:
No, you don't need any scopes at all; the only thing you're doing with the token is asking for the user's UID, GID, and group membership, which doesn't require any scopes. You therefore want:
```
delegate:
  internal:
    service: "nublado"
    scopes: []
```
See https://gafaelfawr.lsst.io/user-guide/gafaelfawringress.html#internal-tokens